sklearn/gaussian_process/gpr.py

```diff
@@ -107,6 +108,11 @@ def optimizer(obj_func, initial_theta, bounds):
         given, it fixes the seed. Defaults to the global numpy random
         number generator.
 
+    n_jobs : int, default: 1
+        n_jobs is the number of workers requested by the callers.
```

Comment:
Please copy the docstring from within scikit-learn, rather than from joblib. This text is a bit obscure.